### DISCUSSION TITLE
Vercel Non-Root Path Deployment

### DISCUSSION THREAD

**Discussion by Southclaws** [vercel/vercel#5233] [Help] [answered]
Hey all!

So we have two sites, `site.com` and `wiki.site.com` - two separate codebases, the site is a Next.js app and the Wiki is a Docusaurus app. Both are hosted on Vercel right now.

What we want to do is move the wiki to `site.com/wiki` for SEO purposes and just general simplicity for users. And of course redirect from `wiki.site.com` to `site.com/wiki` for older links (though the site isn't super old right now, but it's picking up traction so we want to do this quick)

Is this possible with Vercel?

**Comment by styfle** [comment, marked as answer]
Yes this is possible since the release of Monorepos: https://vercel.com/blog/monorepos

You can have two Vercel projects with different root directories connected to the same git repository.

Take a look at the section "How can I make my projects available on different paths under the same domain?"

> After having set up your monorepo as described above, each of the directories will be a separate Vercel project, and therefore be available on a separate domain.
> 
> If you'd like to host multiple projects under a single domain, you can create a new project, assign the domain in the project settings, and proxy requests to the other upstream projects. The proxy can be implemented using a `vercel.json` file with the rewrites property, where each `source` is the path under the main domain and each `destination` is the upstream project domain.

**Reply by Southclaws** [reply]
This sounds like what we've done already, just without the monorepo part. It seems to work fine for now. Though one issue is 404s on the main site get kicked back to the docs site. This is because both sites want to load static assets from `/`. I can't run one site on a basepath because that breaks Vercel previews (which are actually quite important for PR reviews).

It's fine though, we'll figure it out, or just host one site on another service and use Traefik to do rewrites.
